lib/Doctrine/ODM/MongoDB/Aggregation/Expr.php

```diff
@@ -1655,4 +1657,387 @@ public function then($expression): self
+
+    /**
+     * Returns boolean true if the specified expression resolves to an integer, decimal, double, or long.
+     * Returns boolean false if the expression resolves to any other BSON type, null, or a missing field.
+     *
+     * @see https://docs.mongodb.com/manual/reference/operator/aggregation/isNumber/
+     *
+     * @param mixed|self $expression
+     */
+    public function isNumber($expression): self
+    {
+        return $this->operator('$isNumber', $expression);
+    }
+
+    /**
+     * Converts value to a boolean.
+     *
+     * @see https://docs.mongodb.com/manual/reference/operator/aggregation/toBool/
+     *
+     * @param mixed|self $expression
+     */
+    public function toBool($expression): self
+    {
+        return $this->convert($expression, 'bool');
+    }
+
+    /**
+     * Converts value to a Date.
+     *
+     * @see https://docs.mongodb.com/manual/reference/operator/aggregation/toDate/
+     *
+     * @param mixed|self $expression
+     */
+    public function toDate($expression): self
+    {
+        return $this->convert($expression, 'date');
+    }
+
+    /**
+     * Converts value to a Decimal128.
+     *
+     * @see https://docs.mongodb.com/manual/reference/operator/aggregation/toDecimal/
+     *
+     * @param mixed|self $expression
+     */
+    public function toDecimal($expression): self
+    {
+        return $this->convert($expression, 'decimal');
+    }
+
+    /**
+     * Converts value to a double.
+     *
+     * @see https://docs.mongodb.com/manual/reference/operator/aggregation/toDouble/
+     *
+     * @param mixed|self $expression
+     */
+    public function toDouble($expression): self
+    {
+        return $this->convert($expression, 'double');
+    }
+
+    /**
+     * Converts value to an integer.
+     *
+     * @see https://docs.mongodb.com/manual/reference/operator/aggregation/toInt/
+     *
+     * @param mixed|self $expression
+     */
+    public function toInt($expression): self
+    {
+        return $this->convert($expression, 'int');
+    }
+
+    /**
+     * Converts value to a long.
+     *
+     * @see https://docs.mongodb.com/manual/reference/operator/aggregation/toLong/
+     *
+     * @param mixed|self $expression
+     */
+    public function toLong($expression): self
+    {
+        return $this->convert($expression, 'long');
+    }
+
+    /**
+     * Converts value to an ObjectId.
+     *
+     * @see https://docs.mongodb.com/manual/reference/operator/aggregation/toObjectId/
+     *
+     * @param mixed|self $expression
+     */
+    public function toObjectId($expression): self
+    {
+        return $this->convert($expression, 'objectId');
+    }
+
+    /**
+     * Converts value to a string.
+     *
+     * @see https://docs.mongodb.com/manual/reference/operator/aggregation/toString/
+     *
+     * @param mixed|self $expression
+     */
+    public function toString($expression): self
+    {
+        return $this->convert($expression, 'string');
+    }
+ }
```

Comment:
There's an extra space here that phpcs is complaining about:
```suggestion
}
```